test/core/load_balancing/outlier_detection_test.cc

```diff
@@ -197,8 +198,21 @@ TEST_F(OutlierDetectionTest, Basic) {
 }
 
 TEST_F(OutlierDetectionTest, FailurePercentage) {
+  if (!IsEventEngineClientEnabled()) GTEST_SKIP() << "Needs event engine";
```

Comment:
Why is this needed?  I don't think it should be, because this test should not run without event engine enabled.

Same thing in all tests.